llarp/handlers/tun.cpp

```diff
@@ -1074,7 +1074,12 @@ namespace llarp
           }
```

Comment:
I think there should be a `break;` two lines above this (but github won't let me comment there), otherwise we'd get `!allow` if the first one allows and the second one doesn't.